trillian/trampoline/trampoline.go

```diff
@@ -36,7 +37,13 @@ func main() {
 		}
 	}
 
-	fmt.Println(args)
-
-	syscall.Exec(*target, args, os.Environ())
+	for {
```

Comment:
The DB takes a few seconds to start; the Go servers will exit if they don't find their dependencies.